tests/TestHelpers/GraphHelper.php

```diff
@@ -2293,4 +2293,29 @@ public static function getPermissionRoleDefinition(
 			self::getRequestHeaders()
 		);
 	}
+
+	/**
+	 * @param string $baseUrl
+	 * @param string $requestId
+	 * @param string $user
+	 * @param string $password
+	 * @param string $resourceId required for the current first iteration(https://github.com/owncloud/ocis/issues/9194)
+	 *
+	 * @return ResponseInterface
+	 */
+	public static function getActivities(
+		string $baseUrl,
+		string $requestId,
+		string $user,
+		string $password,
+		string $resourceId
+	): ResponseInterface {
+		$fullUrl = self::getBetaFullUrl($baseUrl, "extensions/org.libregraph/activities?kql=itemid%3A$resourceId");
```

Comment:
```suggestion
	): ResponseInterface {
		// itemid filter is required for the current implementation but it might change in the future
		// See: https://github.com/owncloud/ocis/issues/9194
		$fullUrl = self::getBetaFullUrl($baseUrl, "extensions/org.libregraph/activities?kql=itemid%3A$resourceId");
```